pkg/api/exp/flag.go

```diff
@@ -0,0 +1,97 @@
+package exp
+
+import (
+	"strconv"
+	"time"
+)
+
+const (
+	FFPrefix = "feature.dynatrace.com/"
```

Comment:
```suggestion
	FFDynatracePrefix = "feature.dynatrace.com/"
```
or if you saying this prefix is everywhere across codebase? AFAIK we have few others  don't we?

Reply:
this is the "only" FF prefix as of right now, there is 1 oneagent, but that is an outlier, I would leave this as is for now